sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java

```diff
@@ -153,17 +152,26 @@ public void beginPurchaseandReleasePhoneNumbers(HttpClient httpClient) {
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
     public void beginUpdatePhoneNumberCapabilities(HttpClient httpClient) {
         String phoneNumber = getTestPhoneNumber(PHONE_NUMBER);
+
         StepVerifier.create(
-            beginUpdatePhoneNumberCapabilitiesHelper(httpClient, phoneNumber, "beginUpdatePhoneNumberCapabilities").last()
-            .flatMap((AsyncPollResponse<PhoneNumberOperation, PurchasedPhoneNumber> result) -> {
-                assertEquals(LongRunningOperationStatus.SUCCESSFULLY_COMPLETED, result.getStatus());
-                return result.getFinalResult();
-            })
+            this.getClientWithConnectionString(httpClient, "getPurchasedPhoneNumberForCapabilities").getPurchasedPhoneNumberWithResponse(phoneNumber)
+                .flatMap(responseAcquiredPhone -> {
+                    PhoneNumberCapabilities capabilities = new PhoneNumberCapabilities();
+                    capabilities.setCalling(responseAcquiredPhone.getValue().getCapabilities().getCalling() == PhoneNumberCapabilityType.INBOUND ? PhoneNumberCapabilityType.OUTBOUND : PhoneNumberCapabilityType.INBOUND);
+                    capabilities.setSms(responseAcquiredPhone.getValue().getCapabilities().getSms() == PhoneNumberCapabilityType.INBOUND ? PhoneNumberCapabilityType.OUTBOUND : PhoneNumberCapabilityType.INBOUND);
+                    return beginUpdatePhoneNumberCapabilitiesHelper(httpClient, phoneNumber, "beginUpdatePhoneNumberCapabilities", capabilities)
+                        .last()
+                        .flatMap((AsyncPollResponse<PhoneNumberOperation, PurchasedPhoneNumber> result) -> {
+                            assertEquals(LongRunningOperationStatus.SUCCESSFULLY_COMPLETED, result.getStatus());
```

Comment:
We also want to check on the status code of the operation and make sure that it is 200

Reply:
Just checked the PhoneNumberOperationStatus to be success.